templates/assets/js/iframeResize.js

```diff
@@ -23,7 +23,7 @@ window.onmessage = (e) => {
     if (e.data.hasOwnProperty("frameHeight")) {
 	var iframeDiv = document.querySelector("iframe");
 	if (iframeDiv) {
-	    iframeDiv.style["padding-top"] = `${e.data.frameHeight}px`;
+	    // iframeDiv.style["padding-top"] = `${e.data.frameHeight}px`;
```

Comment:
If I remember correctly, this padding was required for the content of the iframe to show up. Have you tested to see that everything still works?

Reply:
Everything which I tested was working fine, I am unable to recollect why I did this change. If I revert back still everything works. But I do remember there was some glitch, hence to debug it I commented this out. But I am not getting it right now.

Reply:
Right now it seems more logical to uncomment this. Maybe the author intended to do something else that we might be skipping out(maybe to handle iframes' height within simulation). Also, right now it is not affecting the layout even on uncommenting.

Reply:
This was done to give the iFrame some height because otherwise it doesn't show anything.